### ISSUE TITLE
Is it possible to retrieve the buffer of an image?

### ISSUE THREAD

**Issue by joliejuke-orga** [meltingice/psd.js#101]
Hi,

This is possible to save an image as PNG but is it possible to retrieve the buffer of an image?

**Comment by fabulousduck** [comment]
Hey @joliejuke-orga. This is possible by looking at the `file` object in any node object.

the following snippet assigns the buffer to the `buf` variable where `node` is the node you would like the buffer from.

`const buf = node.layer.file.data`

**Comment by joliejuke-orga** [comment]
Hey @fabulousduck 
Thanks, I'll try it soon :)

I think one day I must take time to make a more robust documentation about how to parse each property of a psd layer for futur users.

**Comment by joliejuke-orga** [comment]
And do you know if this is possible also to get the font data?
I mean export it as a file or a buffer? 
I think we can only get the font family name, maybe for font licence concerns.

**Comment by joliejuke-orga** [comment]
@fabulousduck 
The buffer contained in the buf variable contain only the image data or the whole file data?

**Comment by fabulousduck** [comment]
@joliejuke-orga 

> Can i get font data from a file ?

No, this is because the reason you stated, and the fact that fonts can have many different variations and would just clutter up the file.

> does buf contain the image data of the whole file ?

Image data of the node/layer `node`


Also for the doc part, if you are willing to do that, I suggest opening a new issue and proposing it there so we can keep track of that convo.
I would however hold of writing such doc for now as @meltingice is working on a new vanillaJS version in wich quite a few things have changed
